docs/lcgs/working-examples/Gossipping_Girls_Circular/can_guarantee_all_girls_eventually_become_omniscient_but_not_girl_one.json

```diff
@@ -0,0 +1,31 @@
+{
+  "enforce eventually": {
+    "players": [1,2,3],
+    "pre": {
+      "and": [
+        {
+          "and": [
+            {
+              "propersition": 2
+            },
+            {
+              "propersition": 3
+            }
+          ]
+        },
+        {
+          "and": [
+            {
+              "propersition": 4
+            },
+            {
+              "not": {
+                "propersition": 3
+              }
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
```

Comment:
`formula` i stedet for `pre`. Og du tjekker både "proposition 3" og "not proposition 3"

Reply:
fixd in 5a1c726c790f7287e6fe4b586f28e54c15294b73 and 75847f436a68a84228a0f2f4cd1f8389e8126d66